lib/gtfs/hastus/activity.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Gtfs.Hastus.Activity do
+  alias Gtfs.{Block, Csv, Run}
+  alias Gtfs.Hastus.{Place, Schedule}
+
+  @type t :: %__MODULE__{
+          schedule_id: Schedule.id(),
+          run_id: Run.id(),
+          start_time: String.t(),
```

Comment:
For stop times we parse times into `time_of_day` values when importing data. Should we do that here too?